docs/sdk-guide.md

```diff
@@ -429,6 +429,15 @@ public CarbonWriterBuilder withAvroInput(org.apache.avro.Schema avroSchema);
 public CarbonWriterBuilder withJsonInput(Schema carbonSchema);
 ```
 
+```
+/**
+* To support writing the ApplicationName which is writing the carbondata file
+* @param application name which is writing the carbondata files
+* @return CarbonWriterBuilder
```

Comment:
mention that it is mandatory method to call

Reply:
this is mandatory because SDK can be used by different application, one application can write and other can read this, so we store this info, during load case we take from spark application name, SDK needs to provide this